cli/smartcontract/smart_contract.go

```diff
@@ -148,26 +148,28 @@ func NewCommands() []cli.Command {
 			{
 				Name:      "invokefunction",
 				Usage:     "invoke deployed contract on the blockchain",
-				UsageText: "neo-go contract invokefunction -r endpoint -w wallet [-a address] [-g gas] scripthash [method] [arguments...] [--] [cosigners...]",
+				UsageText: "neo-go contract invokefunction -r endpoint -w wallet [-a address] [-g gas] scripthash [method] [arguments...] [--] [signers...]",
 				Description: `Executes given (as a script hash) deployed script with the given method,
-   arguments and cosigners. See testinvokefunction documentation for the details
-   about parameters. It differs from testinvokefunction in that this command
-   sends an invocation transaction to the network.
+   arguments and signers. Sender is included in the list of signers by default
+   with FeeOnly witness scope. If you'd like to change default sender's scope, 
+   specify it via signers parameter. See testinvokefunction documentation for 
+   the details about parameters. It differs from testinvokefunction in that this
+   command sends an invocation transaction to the network.
 `,
 				Action: invokeFunction,
 				Flags:  invokeFunctionFlags,
 			},
 			{
 				Name:      "testinvokefunction",
 				Usage:     "invoke deployed contract on the blockchain (test mode)",
-				UsageText: "neo-go contract testinvokefunction -r endpoint scripthash [method] [arguments...] [--] [cosigners...]",
+				UsageText: "neo-go contract testinvokefunction -r endpoint scripthash [method] [arguments...] [--] [signers...]",
 				Description: `Executes given (as a script hash) deployed script with the given method,
-   arguments and cosigners. If no method is given "" is passed to the script, if
-   no arguments are given, an empty array is passed, if no cosigners are given,
-   no array will be passed. All of the given arguments are encapsulated into 
-   array before invoking the script. The script thus should follow the regular
-   convention of smart contract arguments (method string and an array of other 
-   arguments).
+   arguments and signers (sender is not included by default). If no method is given
+   "" is passed to the script, if no arguments are given, an empty array is 
+   passed, if no signers are given no array is passed. All of the given 
```

Comment:
And when there are some signers, the first one of them is going to be the sender, it should be specified here too.